docs/modules/ROOT/pages/reference/extensions/quartz.adoc

```diff
@@ -38,3 +38,60 @@ Or add the coordinates to your existing project:
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Usage
+
+=== Clustering
+
+There are two options how to run Quartz in clustered mode:
+
+==== Quarkus based
+
+Follow the https://quarkus.io/guides/quartz[scheduling periodic tasks quartz guide].
```

Comment:
I think we should make this the preferred configuration option given that it's more optimised. WDYT?

```suggestion
This is the preferred option. Follow the https://quarkus.io/guides/quartz[scheduling periodic tasks quartz guide].
```